src/metrics/runner.py

```diff
@@ -118,12 +120,12 @@ def _inspect_file_sizes(fs: shared_fs.SharedFilesystem) -> tuple[Path, ...]:
 
 
 def _extract_metrics_from_fs_file(
```

Comment:
Since the `fs` args are being renamed, would it be good to rename the `fs` in the method names?